.circleci/config.yml

```diff
@@ -133,6 +133,23 @@ jobs:
       - run: yarn lint
       - run: yarn test
 
+  docs:
+    docker:
+      - image: circleci/python:latest
+    steps:
+      - checkout
+      - setup_remote_docker
```

Comment:
This shouldn't be needed. We need this for waffle tests, because waffle runs solc on docker